### PR TITLE
Create fa_IR.tsx

### PR DESCRIPTION
**Pull request by farskid**
Added fa_IR locale for Persian language.

### CODE REVIEW

**Comment by farskid** [comment]
Seems like there is a deep dependency between locales of ant-design and other packages such as rc-pagination.
Should I update locales of all packages and components at once?

**Comment by afc163** [comment]
Yep, like https://github.com/react-component/pagination/pull/88 and https://github.com/react-component/calendar/pull/264 did.

**Comment by farskid** [comment]
@afc163 Thanks for the rapid reply. I'll update the PR soon :)

**Comment by benjycui** [comment]
ping~ @farskid

**Comment by farskid** [comment]
@benjycui Sorry for the delay, I'll finish it today :)

**Comment by farskid** [comment]
@benjycui I sent PR to pagination, calendar and m-datepicker. pagination and calendar have been merged and released. What's the next step?

**Comment by afc163** [comment]
Check the travis log and make sure it pass.

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/5529

File: components/locale-provider/fa_IR.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import moment from 'moment';
+moment.locale('fa');
+
+import Pagination from 'rc-pagination/lib/locale/fa_IR';
+import DatePicker from '../date-picker/locale/fa_IR';
+import TimePicker from '../time-picker/locale/fa_IR';
+import Calendar from '../calendar/locale/fa_IR';
```
Comment (benjycui) [inline components/locale-provider/fa_IR.tsx]:
```jsx
import DatePicker from '../date-picker/locale/fa_IR';
import TimePicker from '../time-picker/locale/fa_IR';
import Calendar from '../calendar/locale/fa_IR';
```

There are missing.

**Review by afc163 — CHANGES_REQUESTED** [review]
ci breaks

**Comment by benjycui** [comment]
Close for no response.

**Pull request closed without merging**